FormControlRange/explainer.md

```diff
@@ -518,20 +522,20 @@ This `setFormControlRange()` method would set a new flag `IsFormControl()` in th
 
 ```javascript
 range.startContainer        // Returns startNode
-formRange.startContainer    // Flag is on: Throws exception
+formRange.startContainer    // Returns the host <textarea>/<input> element
 ```
 
-This approach addresses encapsulation concerns for ranges in form controls. It also offers better compatibility than the `FormControlRange` interface, since it would work seamlessly with APIs that involve `Range` (not only `AbstractRange`) objects, such as the Selection API:
+This approach addresses encapsulation concerns for ranges in form controls. It also offers better compatibility than the `FormControlRange` interface, since it remains a `Range` and is compatible at the type level with APIs that involve `Range` (not only `AbstractRange`) objects, such as the Selection API. However, Selection behavior for value-space ranges is not defined today and would require additional spec work.
 
 ```javascript
 // Using this approach
-selection.addRange(formRange)     // Works as intended.
+selection.addRange(formRange)     // Type-checks; actual behavior would need explicit spec rules
 
 // Using FormControlRange interface
 selection.addRange(formRange2)    // Throws exception: "parameter 1 is not of type Range."
 ```
 
-Ultimately, the `FormControlRange` interface was chosen instead. While the `setFormControlRange()` approach offered better API compatibility, it did not provide an explicit distinction between regular and form control ranges—both would be of type `Range`, potentially confusing web authors about why some methods may or may not be available depending on how the `Range` was set (using `setStart()` and `setEnd()` or `setFormControlRange()`).
+Ultimately, the `FormControlRange` interface was chosen instead. While the `setFormControlRange()` approach offered better type-level compatibility, it did not provide an explicit distinction between regular and form control ranges—both would be of type `Range`, potentially confusing web authors about why some methods may or may not be available depending on how the `Range` was set (using `setStart()` and `setEnd()` or `setFormControlRange()`), and it would still require additional spec work to define Selection behavior for value-space ranges.
```

Comment:
I would recommend against the addition of "and it would still require additional spec work to define Selection behavior for value-space ranges" here. I think the point about web author confusion is stronger, and the disadvantage of us needing to do more spec work might not be taken as sympathetically and might weaken the overall argument.